src/Header/Header.tsx

```diff
@@ -3,11 +3,16 @@ import { Link } from 'react-router-dom';
 import './Header.scss';
 
 interface HeaderProps {
-	loggedIn: boolean
+	loggedIn: boolean;
+	setLoggedIn: any;
```

Comment:
we should probably figure out the type we need to assign to setLoggedIn rather than any to make it safely typed

Reply:
^^ Agree with Becca's comment above